src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -70,6 +70,9 @@ private static void setupDrivers(Application application) {
 
 		driver = new LineDriverAdapter(drawerController, LineFactory.getSpecialLine(), "special");
 		DriverFeature.addDriver("Special line Simulator", driver);
+
+		DriverFeature.addDriver("Macro recording", MacroFeature.getMacroDriver());
+
```

Comment:
macro should be recorded in background as actual drivers work. Now we cannot record and see the effects of operations on the driver. There are two options:
- you can redesign MacroDriver to decorator pattern
- you can take advantage of driver composite feature and addDriver through AdditionalDriverFeature.

Reply:
Ta funkcjonalność istnieje, mamy dwie opcje skorzystania z MacroDriver, 
jedna to uruchomienie go jako podstawowy sterownik, co spowoduje nagranie makra z wywoływanych poleceń bez żadnej innej akcji;
druga - wybrać "Macro recording" z  "Additional Driver", tak długo ja ta opcja będzie aktywna makro będzie nagrywane, niezależnie od wybranego głównego sterownika